pkg/api/secretbootstrap/secretboostrap.go

```diff
@@ -19,7 +21,7 @@ const (
 )
 
 type ItemContext struct {
-	Item                 string                 `json:"item"`
+	Item                 string                 `json:"item,omitempty"`
```

Comment:
This is technically correct why?

Reply:
When `DockerConfigJSONData` is present there will not be an `Item`. Before I made this change it was outputting `null` in the Marshal process.

Reply:
OK, cool - that's another good validation. One of the two must be set.

Reply:
Evidently a corresponding validation for this exists in `cmd/ci-secret-bootstrap/main.go`. I am wondering if I should pull this validation into this file and remove the existing one?